site/content/en/docs/concepts/multikueue.md

```diff
@@ -7,7 +7,7 @@ description: >
 ---
 
 {{% alert title="Warning" color="warning" %}}
-_Available in Kueue v0.6.0 and later_
+_Available in Kueue v0.7.0 and later_
```

Comment:
Uhm, I agree with your concern, but we already provided feature and featureGate in v0.6.0..
@alculquicondor WDYT?

Reply:
The main issue is that in 0.6.x we didn't support the managedBy mechanism, and for that reason users couldn't install the Jobset controller on management, so the documentation will differ significantly for 0.6 and 0.7.

Reply:
I see the interest in simplicity of the documentation, but it's not true.

Side topic: Let's not user _italic_ if we are already putting a Note. Also, why is it a Warning? Note should be ok.

Reply:
Yeah, it wasn't true. I have reverted the change. PTAL at the update.